src/ol/expr/cpu.js

```diff
@@ -43,6 +44,7 @@ export function newEvaluationContext() {
     properties: {},
     resolution: NaN,
     featureId: null,
+    geometryType: null,
```

Comment:
It looks like the code below uses `''` as the zero value for geometry type (in `computeGeometryType`).  I guess TypeScript doesn't mind, but I think we should either initialize this to `''` or add `null` to the list of accepted types and use `null` to mean unset/unknown.  (I think using `''` is fine for this.)

Reply:
I changed it to `''` for consistency